java/client/src/org/openqa/selenium/remote/http/netty/NettyClient.java

```diff
@@ -94,7 +95,9 @@ private static AsyncHttpClient createHttpClient(ClientConfig config) {
         .setNettyTimer(TIMER)
         .setRequestTimeout(toClampedInt(config.readTimeout().toMillis()))
         .setConnectTimeout(toClampedInt(config.connectionTimeout().toMillis()))
-        .setReadTimeout(toClampedInt(config.readTimeout().toMillis()));
+        .setReadTimeout(toClampedInt(config.readTimeout().toMillis()))
+        .setUseProxyProperties(true)
+        .setUseProxySelector(true);
     return Dsl.asyncHttpClient(builder);
```

Comment:
@pujagani How this will work if user setting Proxy host and port through the ClientConfig, should we set proxy server like below

` .setProxyServer(new ProxyServer.Builder("config.proxy().address()",11).build())`

Reply:
The parameter above is enabled to allow Java proxy properties. Your concern is right. Setting proxy server details via client config does not have an effect currently due to static initialization of the HTTP client https://github.com/SeleniumHQ/selenium/blob/cb6bc2afe413d863c060eb5ca091633e68bcbf1c/java/src/org/openqa/selenium/remote/http/netty/NettyClient.java#L50. To accommodate reading values from client config and initializing on startup is not a trivial change (I did look into it earlier). I will appreciate if you can create a GitHub issue so we can keep track and prioritize accordingly.

Reply:
@pujagani Here is the github issue 
https://github.com/SeleniumHQ/selenium/issues/10231

Reply:
I have already pushed a fix. This will be available in the next release.

Reply:
@diemol thank you so much.

Reply:
Thank you Diego!

Reply:
> I have already pushed a fix. This will be available in the next release.

Could you please let us know when it will be available. I am facing same issue